src/main/java/com/drtshock/playervaults/listeners/Listeners.java

```diff
@@ -54,7 +54,7 @@ public void saveVault(Player player, Inventory inventory) {
             Inventory inv = Bukkit.createInventory(null, 6 * 9);
             inv.setContents(inventory.getContents().clone());
 
-            if (inventory.getViewers().size() == 1) {
+            if (inventory.getViewers().size() >= 1) {
```

Comment:
This has to be 1 or it'll cause issues if someone else is still viewing the vault when a player closes the inventory.